### PR TITLE
Add GhostCell paper

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -3323,3 +3323,13 @@ @inproceedings{bacon04unified
   month=Oct,
   year={2004}
 }
+
+@article{yanovski21ghostcell,
+  title={Ghostcell: separating permissions from data in {Rust}},
```
Comment (ltratt) [inline softdev.bib:3328]: I think this should be `{{GhostCell}}: ...`. Please force push an update.

**Comment by jacob-hughes** [comment]
I've force pushed an update.

**Comment by jacob-hughes** [comment]
I don't know why but adding extra nested `{}` brackets fixes this: 9b78426972c0227b0a231c32b46d40fc503b7ac3

**Comment by ltratt** [comment]
Hmm, I think that will cause problems with bib formats that use title casing. I would personally prefer to do:

```
title="{GhostCell}: separating permissions from data in {Rust}",
```

which I *think* should work.

**Comment by jacob-hughes** [comment]
That doesn't work for some reason. This does though:

```
title="{{GhostCell}: separating permissions from data in {Rust}}",
```

**Comment by ltratt** [comment]
@jacob-hughes Let's try the `"..."` version and we'll see how we get on later. [I must admit that I'm at a loss to explain why the fewer-curlies version doesn't work!]

**Comment by jacob-hughes** [comment]
Ok, fixed here 10630534be2e75fa1da6620970aea80e8d55b6ba

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**